lib/taiko.js

```diff
@@ -638,6 +638,23 @@ module.exports.deleteCookies = async (cookieName, options = {}) => {
   }
 };
 
+/**
+ * Resize the browser window
+ *
+ * @example
+ * await resizeWindow({width:600, height:800})
+ *
+ * @returns {Promise<void>}
+ */
+module.exports.resizeWindow = async (options = {}) => {
+  validate();
+  if (options.height === undefined || options.width === undefined) {
+    throw new Error('No height and width provided');
+  }
+  const [{ targetId }] = await targetHandler.getFirstAvailablePageTarget();
+  await browserHandler.setWindowBounds(targetId, options.height, options.width);
+  descEvent.emit('success', 'Window resized to height ${ options.height } and width ${ options.width }');
```

Comment:
Changing to javascript template string "`" to print the options.

```suggestion
  descEvent.emit('success', `Window resized to height ${ options.height } and width ${ options.width }`);
```